modules/bootkube/manifests.tf

```diff
@@ -17,6 +17,9 @@ variable "manifest_names" {
     "tectonic-node-controller-operator.yaml",
     "tnc-tls-secret.yaml",
     "ign-config.yaml",
+    "app-version-mao.yaml",
```

Comment:
`mao` is not so far from `Mao`. What about `mao` -> `machine-api-operator`. It's not so long even with the additional chars.